### PR TITLE
Adds Engine 0.8.0 release notes and documentation for features.

### PR DESCRIPTION
**Pull request by zhill**
Includes engine-only docs for: malware scanning, content hints, and image deletion changes.

Signed-off-by: Zach Hill <zach@anchore.com>

Co-authored-by: Zach Hill <zach@anchore.com>
Co-authored-by: Dan Nurmi <nurmi@anchore.com>

### CODE REVIEW

File: content/docs/engine/releasenotes/080.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+image's content manifests. This is accomplished with a specific json file present in the image: ```/anchore_hints.json```. The entries are merged into the analyzer results to augment their findings for
+different content types.
+
+For more information, see: [Content Hints]({{< ref "/docs/engine/general/concepts/images/analysis/content_hints" >}})
+
+### Changed Image Deletion Behavior
+
+In this release, image deletion is now an asynchronous operatin and the image_status property in the image record now has possible states 'active' and 'deleting'. Deletion of an image by API call will
+transition the image record to a 'deleting' status as indicated by the image_status property and will be deleted by asynchronous processes on a duty cycle. This helps balance database load under a high volume
+of delete operations and also makes the client-percieved response time much lower.
```
Comment (alfredodeza) [inline content/docs/engine/releasenotes/080.md:39]:
```suggestion
transition the image record to a 'deleting' status as indicated by the `image_status` property and deleted by asynchronous processes on a duty cycle. The asynchronous process helps balance database load under a high volume
of delete operations and makes the client-perceived response time much lower. 
```

File: content/docs/engine/releasenotes/080.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+
+### New API Revision
+
+The API is updated to version [0.1.15](https://github.com/anchore/anchore-engine/blob/v0.8.0/anchore_engine/services/apiext/swagger/swagger.yaml)
+
+### Added
+
++ Changed image deletion to asynchronous behavior to make API more responsive and throttle db load during image deletes.
++ New dry-run mode for repository scan request to return list of tags that would be scanned without actually scanning or persisting the record. Implements [510](https://github.com/anchore/anchore-engine/issues/510)
```
Comment (alfredodeza) [inline content/docs/engine/releasenotes/080.md:52]:
```suggestion
+ New dry-run mode for repository scan request to return list of tags that would be scanned without scanning or persisting the record. Implements [510](https://github.com/anchore/anchore-engine/issues/510)
```

File: content/docs/engine/releasenotes/080.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
++ Support for a "hints" json file in the image. JSON file to pass additional metadata to augment analyzer findings. Implements [550](https://github.com/anchore/anchore-engine/issues/550)
```
Comment (alfredodeza) [inline content/docs/engine/releasenotes/080.md:53]:
```suggestion
+ Support for a "hints" JSON file in the image. JSON file to pass additional metadata to augment analyzer findings. Implements [550](https://github.com/anchore/anchore-engine/issues/550)
```

File: content/docs/engine/releasenotes/080.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
++ Query parameter filters for GET /images calls to filter by image_status and analysis_status. Implements [561](https://github.com/anchore/anchore-engine/issues/561)
+
+### Improved
+
++ Change image analysis queue processing behavior from first come first served to fair share accross account. Implements [491](https://github.com/anchore/anchore-engine/issues/491)
++ Removes image_to_get property in GET body of /images route, since body in GET operations is not standard behavior. Fixes [562](https://github.com/anchore/anchore-engine/issues/562)
+
+### Fixes 
++ Handle scratch images correctly in files gate behavior. Fixes [503](https://github.com/anchore/anchore-engine/issues/530)
++ Add missing fields in swagger json spec for GET /query/vulnerabilities. Fixes [558](https://github.com/anchore/anchore-engine/issues/558)
```
Comment (alfredodeza) [inline content/docs/engine/releasenotes/080.md:66]:
```suggestion
+ Add missing fields in swagger JSON spec for GET /query/vulnerabilities. Fixes [558](https://github.com/anchore/anchore-engine/issues/558)
```

File: content/docs/engine/releasenotes/080.md [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
++ Adds API support for deleting multiple images in a single call. Implements [502](https://github.com/anchore/anchore-engine/issues/502)
++ Support for malware scanning using ClamAV and new 'malware' content type in API and policy gate to trigger on findings. Disabled by default. Implements [498](https://github.com/anchore/anchore-engine/issues/498)     
++ Support for content type 'binary' with analyzers to detect specific binaries: python, golang, busybox not installed by package manager. Implements [531](https://github.com/anchore/anchore-engine/issues/531), [533](https://github.com/anchore/anchore-engine/issues/533), [534](https://github.com/anchore/anchore-engine/issues/534), [340](https://github.com/anchore/anchore-engine/issues/340)
++ Query parameter filters for GET /images calls to filter by image_status and analysis_status. Implements [561](https://github.com/anchore/anchore-engine/issues/561)
+
+### Improved
+
++ Change image analysis queue processing behavior from first come first served to fair share accross account. Implements [491](https://github.com/anchore/anchore-engine/issues/491)
```
Comment (alfredodeza) [inline content/docs/engine/releasenotes/080.md:61]:
```suggestion
+ Change image analysis queue processing behavior from first come first served to fair share across account. Implements [491](https://github.com/anchore/anchore-engine/issues/491)
```

**Comment by zhill** [comment]
@alfredodeza thanks for the review. I'm adding most of that feeback and will update.

**Pull request merged**